src/assets/scss/_table-header.scss

```diff
@@ -28,4 +28,10 @@
     padding-bottom: 2px;
     margin-right: 4px;
   }
+
+  .pull-right {
```

Comment:
pull-right was a bootstrap3 built-in class and that's why we've used it there. Let's name this something more specific. Something like `chaise-table-header-buttons`